contrib/init/bitcoind.service

```diff
@@ -14,8 +14,7 @@ Description=Bitcoin daemon
 Documentation=https://github.com/bitcoin/bitcoin/blob/master/doc/init.md
 
 # https://www.freedesktop.org/wiki/Software/systemd/NetworkTarget/
-After=network-online.target
-Wants=network-online.target
+After=network.target
```

Comment:
If this change finds reviewers' support, isn't cleaner to just revert d9392b724cae53b7a16fa5f84ebe152eea496502?